source/manual/document-collection-emails-a-special-case.html.md

```diff
@@ -0,0 +1,72 @@
+#### How to check if a document_collection has previously been published
+
+1. Find the document_collection in the whitehall admin UI
+2. Find the ID. E.g. `12345` for a document_collection at URL `whitehall-admin.publishing.service.gov.uk/government/admin/collections/12345`
+3. `DocumentCollection.find(12345).document.live?` returns `true` for previously published document_collections.
+
+#### How to set a taxonomy topic email override field for new pages
+
+This field can be set by running the following [rake task][whitehall_rake_task] from a Whitehall machine. This task runs in dry_run mode by default, unless a confirmation string is passed in:
+
+DRY RUN:
+>`rake:set_taxonomy_topic_email_override[document_collection_id,taxon_content_id]`
+
+FOR REAL:
+>`rake:set_taxonomy_topic_email_override[document_collection_id,taxon_content_id,run_for_real]`
+
+[whitehall_rake_task]: https://github.com/alphagov/whitehall/blob/main/lib/tasks/set_taxonomy_topic_email_override.rake
+
+#### How to set a taxonomy topic email override field for previously published pages
+
+It is not possible to set this field on a previously published page. If a request is made to `2nd line - GOV.UK content` to add this feature to a live page, their documentation aligns with the guidance below:
+
+- The department must create a new draft replacement document collection.
+- Developer sets a taxonomy topic email override on the replacement document by following the steps in the section [How to set a taxonomy topic email override field for new pages][heading]
+- Department checks the email settings in the Whitehall UI and then publishes the page.
+- Content designer unpublishes the original page and creates a redirect from the old document collection to the new document collection.
+
+**Note:** there will then need to be some clearup of email subscribers who signed up whilst the original page was live:
+
+##### If the taxonomy_topic_email_override on the original page was nil
+
+In this case, users would have subscribed to a content_id based list for the original document collection. Those users can be migrated to the appropriate taxonomy topic SubscriberList. See [documentation][coming-soon] in email-alert-api on how to bulk migrate email subscribers.
+
+##### The taxonomy_topic_email_override on the original page was not nil
+
+In this case, users would have subscribed to a taxonomy topic SubscriberList. Email alert API does not store information about the page from which a user subscribed to a links-based SubscriberList. So it is not possible to confidently identify the users that need to be migrated. This limitation/risk was acknowledged by both the department, and senior leadership when the feature was rolled out. Therefore this scenario is highly unlikely, and is included here just for completeness.
+
+[heading]: [#how-to-set-a-taxonomy-topic-email-override-field-for-new-pages]
+[coming-soon]: https://github.com/alphagov/email-alert-api/blob/main/lib/tasks/coming-soon.rake
```

Comment:
This doesn't exist at all! What is this? 😱 
Should it link here: https://github.com/alphagov/email-alert-api/blob/main/lib/tasks/data_migration.rake#L5 ?